src/ui/src/components/blueprints/BlueprintsAutogenContents.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div class="BlueprintsAutogenContents">
+		<div class="BlueprintsAutogenContents__contents">
+			<slot></slot>
+		</div>
+		<div class="BlueprintsAutogenContents__background">
+			<GradientCircle class="rotating-circle left" />
+			<GradientCircle class="rotating-circle center" />
+			<GradientCircle class="rotating-circle right" />
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import GradientCircle from "./GradientCircle.vue";
+</script>
+
+<style scoped>
+.BlueprintsAutogenContents {
+	position: relative;
+	height: 306px;
+	overflow: hidden;
+	width: 100%;
+	border-radius: 8px;
+}
+
+.BlueprintsAutogenContents__contents {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	padding: 40px;
+	z-index: 10;
+	width: 100%;
+	height: 100%;
+}
+
+.BlueprintsAutogenContents__background {
+	position: "absolute";
+	inset: 0;
+	background: linear-gradient(0deg, #ffd5f8 0.01%, #bfcbff 99.42%);
+	opacity: 50%;
+}
+
+.BlueprintsAutogenContents.busy .BlueprintsAutogenContents__background {
+	opacity: 100%;
+}
```

Comment:
It's a bit weird to have `busy` class coming from the parent and the CSS rule here.

I suggest you bring down the prop `isBusy` here.